docs/sources/whatsnew/whats-new-in-v9-1.md

```diff
@@ -0,0 +1,250 @@
+
+Now, you can rotate keys and re-encrypt secrets via API.
+This makes it easier to configure Grafana to be secure while deploying it.
+To learn more, see our guide to [configuring database encryption]({{< relref "../setup-grafana/configure-security/configure-database-encryption/#configure-database-encryption" >}}).
+
+#### Audit all actions in Grafana using verbose logging
+
+Auditing logs helps you manage and mitigate activity and meet compliance requirements.
+By default, Grafana emits an audit log with every action that changes something (like a user creating or updating a dashboard or updating another user's permissions).
+If you want to record all actions on the Grafana server, including GETs and page views, you can now turn on `verbose mode`.
+This results in more logs, but it can be useful to debug specific issues or make sure you catch everything happening in Grafana for security or compliance reasons.
+To learn more, see the [configuration documentation]({{< relref "../setup-grafana/configure-grafana/enterprise-configuration/#verbose" >}}).
+
+{{< figure src="/static/img/docs/enterprise/verbose-audit-logs-9-1.png" max-width="750px" caption="Verbose audit logging output" >}}
+
+#### See (and don't edit) users synced from SAML, LDAP, and OAuth identity providers
+
+When you synchronize users from a SAML, LDAP, or OAuth provider, some user settings, such as name and email address, are synchronized from your identity provider.
+Previously, you could edit those settings in the Grafana UI, but they would revert back.
+To make user management clearer, you can now see which settings are synchronized from your identity provider, but you cannot edit those settings.
+To learn more about authentication, see the [documentation]({{< relref "../setup-grafana/configure-security/configure-authentication/" >}}).
+
+{{< figure src="/static/img/docs/enterprise/oauth-synced-user-9-1.png" max-width="750px" caption="Non-interactive view of a user synced via OAuth" >}}
+
+#### Support wildcards for LDAP groups in team sync
+
+Team sync lets you set up synchronization between your authentication provider's teams and Grafana teams.
+To leverage your existing Active Directory in an efficient way without having to create multiple teams, you can now use wildcards when configuring LDAP groups, so that multiple Active Directory groups can be added at once to a single team.
+
+{{< figure src="/static/img/docs/enterprise/ldap-wildcard-teamsync-9-1.png" max-width="750px" caption="Using wildcard mapping for team sync" >}}
+
+#### Redirect binding support for AzureAD SAML Single Logout
+
+SAML Single Logout allows users to log out from all applications associated with the current IdP (Identity Provider) session established via SAML Single Sign-On (SSO).
+To enable integration of more use cases, we have added support for an HTTP-Redirect binding for a Single Logout.
+
+#### AWS Secrets Manager integration
```

Comment:
Hi, can we drop this line item from the "what's new" section?

Reply:
Dropped in https://github.com/grafana/grafana/pull/53066/commits/66e4876618f2d6faaeea8f43dfba74da88a8cd47